packages/dds/tree/src/events/emitter.ts

```diff
@@ -97,6 +97,20 @@ export interface HasListeners<TListeners extends Listeners<TListeners>> {
  * 	}
  * }
  * ```
+ *
+ * @example Exposing this class as a property
+ *
+ * ```typescript
+ * class MyExposingClass {
+ * 	private readonly _events = createEmitter<MyEvents>();
+ *  public readonly events: Listenable<MyEvents> = this._events;
```

Comment:
```suggestion
 * 	private readonly _events = createEmitter<MyEvents>();
 *  	public readonly events: Listenable<MyEvents> = this._events;
```

Reply:
Ugh - that's weird. Thanks. I'll fix it in the next event PR.